.github/actions/rust-build-release/tests/test_commands.py

```
@@ -0,0 +1,607 @@
+def test_build_cross_command_rejects_injected_toolchain_override(
+    monkeypatch: pytest.MonkeyPatch,
+    tmp_path: Path,
+) -> None:
+    """_build_cross_command raises ValueError when argv would contain +toolchain."""
+    main_module = _load_main_module(monkeypatch)
+    cross_path = _make_cross_executable(tmp_path)
+
+    # Patch the guard to inject a +toolchain token after the fact
+    original_guard = main_module._assert_cross_command_has_no_toolchain_override
+
+    def injecting_guard(cmd: object) -> None:
```

Ruff (0.15.12)

[warning] Unused function argument: `cmd`

(ARG001)